MixpanelReactNative.podspec

```diff
@@ -19,5 +19,5 @@ Pod::Spec.new do |s|
   s.pod_target_xcconfig = { 'DEFINES_MODULE' => 'YES' }  
   
   s.dependency "React-Core"
-  s.dependency "Mixpanel-swift", '4.0.2'
+  s.dependency "Mixpanel-swift", '4.0.5'
```

Comment:
@dneralla can you please try updating `s.platform = :ios, "9.0"` to `"11.0"` to see if that fixes CI?